webextension/scripts/popup.js

```diff
@@ -139,15 +139,28 @@ function search_tweet() {
   })
 }
 
+function makeValidURL() {
+  if (!isValidUrl(searchValue)) {
+    searchValue = 'https://' + searchValue
+  }
+}
+
```

Comment:
Instead of using a global, better to pass in the variable to this function?

Reply:
Nope. That's not the way to do it. Here:

```suggestion
function makeValidURL(url) {
  return isValidUrl(url) ? url : 'https://' + url
}

```

Reply:
Thank you!